### PR TITLE
updating README in preparation for matlab-actions/run-tests@v2

### PR DESCRIPTION
**Pull request by mw-hrastega**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,36 +17,36 @@ jobs:
...
+        uses: actions/checkout@v4
       - name: Run tests
-        uses: matlab-actions/run-tests@v1
+        uses: matlab-actions/run-tests@v2
 ```
 
-### Generate Test Artifacts on GitHub-Hosted Runner
-Before you run tests and generate artifacts on a GitHub-hosted runner, first use the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action. The action sets up your specified MATLAB release (R2020a or later) on a Linux&reg; virtual machine. If you do not specify a release, the action sets up the latest release of MATLAB.
+### Generate Artifacts on GitHub-Hosted Runner
+Before you run tests and generate artifacts on a [GitHub-hosted runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners), first use the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action. The action sets up your specified MATLAB release (R2020a or later) on a Linux&reg;, Windows&reg;, or macOS virtual machine. If you do not specify a release, the action sets up the latest release of MATLAB.
```
Comment (mcafaro) [inline README.md:26]: I believe `setup-matlab@v2` is R2020b or later.

Comment (mcafaro) [inline README.md:26]: macOS is a registered trademark. https://www.apple.com/legal/intellectual-property/trademark/appletmlist.html

File: README.md [hunk 2]
```diff
@@ -1,16 +1,12 @@
...
+The [Run MATLAB Tests](#run-matlab-tests) action enables you to run MATLAB&reg; and Simulink&reg; tests and generate test and coverage artifacts on a self-hosted or GitHub&reg;-hosted runner. The runner uses the topmost MATLAB version on the system path to execute your workflow.
 
 ## Examples
 Use the **Run MATLAB Tests** action to automatically run tests authored using the MATLAB unit testing framework or Simulink Test&trade;. You can use this action with optional inputs to generate various test and coverage artifacts.
 
 ### Run MATLAB Tests on Self-Hosted Runner
-Use a self-hosted runner to automatically run the tests in your [MATLAB project](https://www.mathworks.com/help/matlab/projects.html).
+Use a [self-hosted runner](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners) to run the tests in your [MATLAB project](https://www.mathworks.com/help/matlab/projects.html).
```
Comment (mcafaro) [inline README.md:9]: It would be nice to get the "R2013b or later" back in here somewhere.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,12 @@
...
-
-- To use a self-hosted runner, you must set up a computer with MATLAB (R2013b or later) as your runner. The runner uses the topmost MATLAB version on the system path to execute your workflow.
-
-- To use a GitHub-hosted runner, you must include the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action in your workflow to set up MATLAB on the runner. Currently, this action is available only for public projects. It does not set up transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
+The [Run MATLAB Tests](#run-matlab-tests) action enables you to run MATLAB&reg; and Simulink&reg; tests and generate test and coverage artifacts on a self-hosted or GitHub&reg;-hosted runner. The runner uses the topmost MATLAB version on the system path to execute your workflow.
```
Comment (mcafaro) [inline README.md:3]: "to execute your workflow" This action is not necessarily your entire workflow.

```suggestion
The [Run MATLAB Tests](#run-matlab-tests) action enables you to run MATLAB&reg; and Simulink&reg; tests and generate test and coverage artifacts on a self-hosted or GitHub&reg;-hosted runner. The action uses the topmost MATLAB version on the system path.
```

File: README.md (hunk 2 above)
Comment (mw-hrastega) [reply README.md:9]: With 2012a, there is a 11+ year gap. Do we expect users to still rely on R2012a or earlier, or is it safe to assume that this information only pertains to a very tiny group of CI users?

Comment (mcafaro) [reply README.md:9]: Up to you. I suspect it only pertains to a very tiny group.

File: README.md (hunk 1 above)
Comment (mw-hrastega) [reply README.md]: Updated.

Comment (mw-hrastega) [reply README.md]: Updated. It's strange that MATLAB documentation has been using macOS without any trademark symbols: https://www.mathworks.com/help/matlab/ref/matlabmacos.html?searchHighlight=macOS&s_tid=srchtitle_support_results_1_macOS. Maybe Apple added the symbol recently.

File: README.md (hunk 2 above)
Comment (mw-hrastega) [reply README.md:9]: Ok, then maybe we can remove it and add it back if we receive complaints. 🙂

**Review by mcafaro — APPROVED** [review]

**Review by davidbuzinski — APPROVED** [review]

**Pull request merged**